pkgs/by-name/qw/qwerty-fr/package.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv
+, fetchurl
+, lib
+, unzip
+,
```

Comment:
This should probably either have the trailing comma and be formatted with `pkgs.nixfmt-rfc-style`, or not have the trailing comma